tests/test_psfCandidate.py

```diff
@@ -202,7 +202,13 @@ def testFaintNeighborMasking(self):
 
     def testStr(self):
         candidate = self.createCandidate()
+
+        # The setX methods are class globals and could contain values set
+        # by other code. Force the values here to ensure test consistency.
         candidate.setChi2(2.0)
+        candidate.setWidth(0)
```

Comment:
It might be better if I set everything to non-default values so that we can know for sure we are seeing those values in the string form.

Reply:
No preference either way from me (I didn't even know these values had defaults).